tests/src/installAgents.ts

```diff
@@ -1,78 +1,97 @@
-
-const { Codec } = require("@holo-host/cryptolib");
+const { Codec } = require('@holo-host/cryptolib')
 import * as path from 'path'
-import * as msgpack from '@msgpack/msgpack';
-import { InstalledHapp, Player } from '@holochain/tryorama';
+import * as msgpack from '@msgpack/msgpack'
+import { InstalledHapp, Player } from '@holochain/tryorama'
 
 const dnaConfiguration = {
-  role_id: 'elemental-chat',
+	role_id: 'elemental-chat',
 }
-const dnaPath = path.join(__dirname, "../../elemental-chat.dna")
-const jcFactoryDna = path.join(__dirname, "../../dnas/joining-code-factory.dna");
-const installedAppId = agentName => `${agentName}_chat`
+const dnaPath = path.join(__dirname, '../../elemental-chat.dna')
+const jcFactoryDna = path.join(__dirname, '../../dnas/joining-code-factory.dna')
+const installedAppId = (agentName) => `${agentName}_chat`
 
 export type Memproof = {
-  signed_header: {
-    header: any,
-    signature: Buffer,
-  },
-  entry: any
+	signed_action: {
+		header: any
```

Comment:
Should probably update this to `action` too.
```suggestion
		action: any
```